cpu_power_logging_node:
  ros__parameters:
    logging_interval: 0.1
